setup.py

```diff
@@ -3,19 +3,19 @@
 with open("README.md", encoding="utf-8") as f:
     long_description = f.read()
 
+with open("m2cgen/VERSION.txt") as f:
+    version = f.read().strip()
+
 setup(
     name="m2cgen",
-    version="0.4.1",
+    version=version,
     url="https://github.com/BayesWitnesses/m2cgen",
     description="Code-generation for various ML models into native code.",
     long_description=long_description,
     long_description_content_type="text/markdown",
     license="MIT",
-    packages=find_packages(exclude=["tests.*", "tests"]),
-    package_data={
-        "": ["linear_algebra.java", "linear_algebra.c", "linear_algebra.go",
-             "linear_algebra.js"],
-    },
+    packages=find_packages(exclude=["tests.*", "tests", "tools"]),
```

Comment:
I think that `tools` should be included into `m2cgen`'s installation process. Please correct me if I'm wrong.

Reply:
No, I don't think so, why? `tools` are dev tools and I don't see why they should be a part of the installation process.

Reply:
Oh, I'm so sorry, I forgot to type **not**! I excluded them in code but failed to comment that 😢 .

Reply:
Ah, makes sense now. It only now occurred to me that the comment contradicted the code :)